src/pseudo.rs

```diff
@@ -19,11 +19,11 @@ use crate::{
 fn process_triple(
     triple: Triple,
     rules_config: &Rules,
-    node_to_type: &HashMap<String, String>,
+    index: &HashMap<String, String>,
```

Comment:
Nitpicking: even though it makes sense to use it in our case, the word `index` is often used as a number to access a list element. When I read `match_rules(&triple, rules_config, index)`, my first assumption is that it will try to access the rule at `index` position in the list (`rules_config[index]`) or something similar. IMO `node_to_type` was a bit more clear.

I never really know how to name these. Usually, I think people would use `node_to_type`, `type_of_node` or  `node_type_map`, but there may be a better way :grin: 

If we really want to use the word "index", maybe we could have a name like `types_index` or something that describes a bit better what it does?

Reply:
fair point! I've reversed that in [4859547](https://github.com/sdsc-ordes/tripsu/pull/48/commits/48595472bcdd1ef7aa32b15ba02f7825a5d28fc4)